website/docs/developer/deployment-guide/deploying-from-windows.mdx

```diff
@@ -0,0 +1,185 @@
+- In each terraform.tf and terraform.tfvars, use your own prefix, region, vpc id, and subnet ids.
+  :::warning
+  The region and prefix are not always in simple variables. Do a global search for 'PREFIX' and 'us-east-1'.
+  :::
+  :::warning
+  Only use the non-lambda subnet id in the data-persistence-tf/terraform.tfvars. In cumulus-tf use both.
+  :::
+  :::warning
+  Overwrite the ```orca-sandbox``` in ```orca-sandbox-tf-locks``` with your prefix as well.
+  :::
+- In rds-cluster-tf/terraform.tfvars
+    - Use values of your choice for ```db_admin_username``` and ```db_admin_password```
+    - Set ```tags``` to ```{ "Deployment" = "PREFIX" }```
+    - Set ```permissions_boundary_arn``` to ```arn:aws:iam::YOUR ACCOUNT ID:policy/NGAPShRoleBoundary```
+    - Add ```rds_user_password = "CumulusD3faultPassw0rd"``` and change as desired.
+    - Set ```provision_user_database``` to ```true```
+    - Set ```cluster_identifier``` to ```"PREFIX-cumulus-db"```
+
+- In rds-cluster-tf/terraform.tf
+    - Set ```bucket``` to ```"PREFIX-tf-state"```
+    - Set ```key``` to ```"PREFIX/cumulus/terraform.tfstate"```
+    - Set ```dynamodb_table``` to ```"PREFIX-tf-locks"```
+
+- Go to https://github.com/asfadmin/CIRRUS-core/blob/master/Dockerfile and download the file to the same folder as your downloaded repo and orca folder.
+  :::tip
+  Make sure that no extension is added.
+  :::
+- Open a commandline in the same folder.
+    - Run ```docker build -t orca .``` and ```docker run -it --rm -v pathToYourFolder:/CIRRUS-core orca /bin/bash```
+    - The commandline should now be inside a docker container.
+      ```bash
+      cd cumulus-orca-template-deploy/rds-cluster-tf/
+      aws configure
+      terraform init
+      terraform plan
+      terraform apply
+      ```
+
+- In data-persistence-tf/terraform.tfvars
+    - Set ```permissions_boundary_arn``` to ```arn:aws:iam::12345:policy/NGAPShRoleBoundary``` replacing the ```12345``` with your Account Id.
+      <img alt="Location of Account Id" src={useBaseUrl('img/aws-account-id.PNG')} />
+    - Set ```rds_user_access_secret_arn``` to the ```user_credentials_secret_urn``` output from ```terraform apply```.
+    - Set ```rds_security_group``` to the ```security_group_id``` output from ```terraform apply```.
+    - Set ```vpc_id``` to your borrowed VPC.
+
+```bash
+cd ../data-persistence-tf/
+aws configure
+terraform init
+terraform plan
+terraform apply
+```
+
+- In cumulus-tf/terraform.tfvars
+    - Add the line ```aws_profile = "default"```
+    - Replace 12345 in permissions_boundary_arn with the Account Id.
+    - Add to the buckets:
+      ```
+      default_orca = {
+         name = "prefix-orca"
+         type = "orca"
+        },
+        provider = {
+          name = "orca-sandbox-s3-provider"
+          type = "provider"
+      }
+      ```
+      :::tip
+      The "orca-sandbox-s3-provider" bucket contains test data.
+      If creating a separate environment, you can create your own bucket.
```

Comment:
I would probably recommend just having the 1 provider bucket so we are all pulling from the same set of test data.

Reply:
I would prefer to keep the option in. We should be able to document a fresh install in addition to a hacked-together test environment, and this is one step closer to that.

Reply:
ok, but maybe put some verbiage in that the same test data should be used in each bucket. The test data can be retrieved from the orca-sandbox-s3-provider bucket.

Reply:
Added.